File: java/P02-strings.js

// Practice 02: Problem 1:
let doubleLetters = (strang) => {
  return (((strang.split("")) + (strang.split(""))).join(""))
}
console.log(doubleLetters("word"))

// ----------------------------------------------------------------------------

function double_letter_using_loops(text){
  let double_string = ''
  for (let i=0; i<text.length; i++) {
    double_string += text[i] text [i]
  }
  return double_string
}
// Worked
// ----------------------------------------------------------------------------

const double_letters_using_map = (text) => text.split('')
                                               .map(letter => letter + letter)
                                               .join('') 
// Worked

File: python/P02-strings.py
# Practice 02-Strings.py

# example:


def add(a, b):
    return a + b
# print(add(5, 2))
# print(add(8, 1))

# Problem 1:


def double_letters(x):
    double = ''
    for char in x:
        double += char * 2
    return double


# return ''.join([char*2 for char in text])

print(double_letters('hello'))


#
